src/CMakeLists.txt

```diff
@@ -96,13 +96,19 @@ add_library(gpuarray-static STATIC ${GPUARRAY_SRC})
 target_link_libraries(gpuarray ${CMAKE_DL_LIBS})
 target_link_libraries(gpuarray-static ${CMAKE_DL_LIBS})
 
+get_target_property(GPUARRAY_ABI_VERSION gpuarray VERSION)
+FILE(WRITE gpuarray/abi_version.h
+"\#ifndef GPUARRAY_ABI_VERSION\n\#define GPUARRAY_ABI_VERSION \"${GPUARRAY_ABI_VERSION}\"\n\#endif\n"
```

Comment:
Please make it an int like (major * 1000 + minor).  Having it as a string is just super inconvenient for most uses.